packages/model-serving/src/components/deploymentWizard/fields/ModelServingHardwareProfileSection.tsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react';
+import HardwareProfileFormSection from '@odh-dashboard/internal/concepts/hardwareProfiles/HardwareProfileFormSection';
+import {
+  useHardwareProfileConfig,
+  type UseHardwareProfileConfigResult,
+} from '@odh-dashboard/internal/concepts/hardwareProfiles/useHardwareProfileConfig';
+import { SupportedArea, useIsAreaAvailable } from '@odh-dashboard/internal/concepts/areas/index';
+import { HardwareProfileFeatureVisibility } from '@odh-dashboard/internal/k8sTypes';
+import { ContainerResources, NodeSelector, Toleration } from '@odh-dashboard/internal/types';
+import type {
+  PodSpecOptions,
+  PodSpecOptionsState,
+} from '@odh-dashboard/internal/concepts/hardwareProfiles/types';
+import type { UseAcceleratorProfileFormResult } from '@odh-dashboard/internal/utilities/useAcceleratorProfileFormState.js';
+
+export const useModelServingHardwareProfileSection = (
+  name?: string,
+  resources?: ContainerResources,
+  tolerations?: Toleration[],
+  nodeSelector?: NodeSelector,
+  namespace?: string,
+  hardwareProfileNamespace?: string,
+): UseHardwareProfileConfigResult => {
+  const isProjectScoped = useIsAreaAvailable(SupportedArea.DS_PROJECT_SCOPED).status;
+
+  return useHardwareProfileConfig(
+    name,
+    resources,
+    tolerations,
+    nodeSelector,
+    [HardwareProfileFeatureVisibility.MODEL_SERVING],
+    isProjectScoped ? namespace : undefined,
+    hardwareProfileNamespace,
+  );
+};
+
+export const createAcceleratorProfileStub = (): UseAcceleratorProfileFormResult => ({
```

Comment:
Is this needed because Razz needs to remove the requirement from `<HardwareProfileFormSection />`?